pde_superresolution/integrate_test.py

```diff
@@ -174,6 +174,24 @@ def test_integrate_baseline_and_weno_consistency(self, equation, tol=1e-3):
         results_baseline.drop('num_evals'), results_weno.drop('num_evals'),
         rtol=tol, atol=tol)
 
+  @parameterized.parameters(
+    dict(equation=equations.GodunovBurgersEquation(100),
+         differentiator=integrate.WENODifferentiator),
+    dict(equation=equations.ConservativeBurgersEquation(100),
+         differentiator=integrate.PolynomialDifferentiator),
+    dict(equation=equations.BurgersEquation(100),
+         differentiator=integrate.PolynomialDifferentiator))
+  def test_burgers_symmetry(self, equation, differentiator):
+      def flip_minus(x):
+          return -x[::-1]
+      time = np.random.randn()
+      time_differentiator = differentiator(equation)
+      u = np.random.randn(100)
```

Comment:
Can you make this use a seed, e.g., `np.random.RandomState(0).randn(100)`? That way the test is deterministic and can't fail stochastically.

Reply:
done